Development/Scripts/CanvasScalerFixed.cs

```diff
@@ -5,7 +5,7 @@ namespace Phedg1Studios {
     namespace StartingItemsGUI {
         public class CanvasScalerFixed : CanvasScaler {
             public void ForcedUpdate() {
-                Update();
+                //Update();

```

Comment:
Commented out for now. It didn't seem to be used anyway, and wouldn't compile with it enabled.